package.json

```diff
@@ -359,6 +359,7 @@
     "test-unit-keep-cljs": "jest --maxWorkers=2",
     "test-unit-watch": "yarn test-unit --watch",
     "test-unit-watch:cljs": "yarn concurrently -n 'cljs,tests' 'yarn build-watch:cljs' 'yarn test-unit-keep-cljs --watch'",
+    "test-unit-watch:debug": "node --inspect-brk node_modules/.bin/jest --runInBand --no-cache --watch",
```

Comment:
I haven't tried it, but I am assuming you can't use just `yarn jest` because `--inspect-brk` flag is needed to be passed to node?

Reply:
Have found this though: https://stackoverflow.com/a/48550380
If it works, might be a bit easier to read.

Reply:
We're on an [old version of yarn](https://github.com/metabase/metabase/issues/27199) and the `--inspect` flag is not available.

We could use `NODE_OPTIONS="--inspect"` but I don't think it's necessarily easier to understand/read than `node --inspect-brk path`.